.gitignore

```diff
@@ -251,5 +268,22 @@ paket-files/
 .idea/
 *.sln.iml
 
-# Ignore customized Config.json
-Source/ACE/Config.json
```

Comment:
Yes. From the review comment, it sounded like it was a base `.gitignore` from GitHub, and I didn't bother to check. Sorry.